guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -10,16 +10,23 @@ functional_areas:
   - Setup
   - Configuration
 ---
+The ece-tools package, also known as _ece-tools_, is compatible with Magento version 2.1.4 and later to provide a rich set of features you can use to manage your {{site.data.var.ee}} project.  It contains scripts and {{site.data.var.ece}} commands designed to help manage your code and automatically build and deploy your Cloud projects.
 
-{% include cloud/ece-tools.md %}
+You can list the available ece-tools commands using:
+
+```bash
+php ./vendor/bin/ece-tools list
+```
+
+The following updates describe the latest improvements to the ece-tools package, which update as needed through patching and product upgrades managed by the `magento-cloud-metapackage`. The ece-tools package adheres to the version sequence:  `200<major>.<minor>.<patch>`.
 
 ## v2002.0.12
 
-{% include note.html type="info" content="The Cloud tools version 2002.0.12 now supports Magento 2.1.14. " %}
+{% include note.html type="info" content="The ece-tools version 2002.0.12 now supports Magento 2.1.14. " %}
```

Comment:
I think we should use inline attributes to define notes rather than liquid + includes because the latter doesn't support certain things. For example, I don't think you can use Jekyll variables when you use this format. This could lead to broken things in the future. See [Style Guide](https://wiki.corp.magento.com/display/MDOC/Style+guide+-+formatting#Styleguide-formatting-Links).

Preferred format:

```
{:.bs-callout bs-callout-info}
This is a note.
```